Gaijinizer patch/Map353.txt

```diff
@@ -180,10 +180,10 @@ a scared child on your behalf.#GT
 
 覚悟しなさい！！
 > CONTEXT: Map353/events/1/pages/0/150/Dialogue/enemy
-I have to squeeze you once before going
-to the exit from \\N[1].
+I'll have to squeeze you out before I can take
+\\N[1] to the exit.
 
-Be prepared!!#GT
+Prepare yourself!!#Ron
 > END STRING
```

Comment:
Note?

Reply:
Oh yeah, I don't know why I didn't mark this as CheckLater. I couldn't really make sense of this one.

Reply:
Seems to be Real Lampas talking to fake Lampas. Makes enough sense in context.